unity-renderer/Assets/DCLPlugins/RealmsPlugin/Utils/JumpToHomeController.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using DCL;
+using DCL.Interface;
+using UnityEngine;
+using UnityEngine.UI;
+using Variables.RealmsInfo;
+
+public class JumpToHomeController : MonoBehaviour
+{
+
+    [SerializeField] private Button jumpButton;
+    [SerializeField]private ShowHideAnimator showHideAnimator;
+    [SerializeField] private RectTransform positionWithMiniMap;
+    [SerializeField] private RectTransform positionWithoutMiniMap;
+
+    private BaseCollection<RealmModel> realms => DataStore.i.realm.realmsInfo;
+    private BaseVariable<bool> jumpHomeButtonVisible => DataStore.i.HUDs.jumpHomeButtonVisible;
+    private BaseVariable<bool> minimapVisible => DataStore.i.HUDs.minimapVisible;
+    private RectTransform rectTransform;
+
+    private void Start()
+    {
+        rectTransform = jumpButton.GetComponent<RectTransform>();
+        jumpButton.onClick.AddListener(GoHome);
+        jumpHomeButtonVisible.OnChange += SetVisibility;
+        SetVisibility(jumpHomeButtonVisible.Get(), false);
+    }
+    
+    private void SetVisibility(bool current, bool _)
```

Comment:
nice to see that you started to use `_` for unsure parameters 💪

Reply:
Yeaah